sbi/simulators/linear_gaussian.py

```diff
@@ -4,37 +4,37 @@
 import sbi.utils as utils
 
 
-def linear_gaussian(parameters: torch.Tensor, std=1.0) -> torch.Tensor:
+def linear_gaussian(theta: torch.Tensor, std=1.0) -> torch.Tensor:
 
-    if parameters.ndim == 1:
-        parameters = parameters[None, :]
+    if theta.ndim == 1:
+        theta = theta[None, :]
```

Comment:
I am not sure, this is out of the scope of this PR, but we should think whether we need this check and if yes do it with `unsqueeze(0)`?

Reply:
or using `atleast_2d`?